PatternVariableName/all-examples-in-one/config.xml

```diff
@@ -36,6 +36,30 @@
             <property name="id" value="example3"/>
         </module>
 
+        <module name="PatternVariableName">
+            <property name="format" value="^[A-Z][A-Z0-9]*$"/>
+            <property name="id" value="example4"/>
+            <property name="id" value="example4"/>
```

Comment:
Why twice id ?

Reply:
This should be fixed

Reply:
> This should be fixed

But, i guess the example is written that way. 
https://github.com/checkstyle/checkstyle/blob/f1a36722c5038aa104f7be12486cf2d7edff37ee/src/xdocs-examples/resources-noncompilable/com/puppycrawl/tools/checkstyle/checks/naming/patternvariablename/Example4.java

Reply:
```
    <module name="PatternVariableName">
        <property name="id" value="FinalPatternVariableName"/>
        <property name="format" value="^[A-Z][A-Z0-9]*$"/>
    </module>

    <module name="PatternVariableName">
        <property name="id" value="NonFinalPatternVariableName"/>
        <property name="format" value="^([a-z][a-zA-Z0-9]*|_)$"/>
    </module>

    <module name="SuppressionXpathSingleFilter">
      <property name="id" value="FinalPatternVariableName"/>
      <property name="query" value="//PATTERN_VARIABLE_DEF[
            not(./MODIFIERS/FINAL)]/IDENT"/>
    </module>

    <module name="SuppressionXpathSingleFilter">
      <property name="id" value="NonFinalPatternVariableName"/>
      <property name="query" value="//PATTERN_VARIABLE_DEF[
            (./MODIFIERS/FINAL)]/IDENT"/>
```

Reply:
All these are multiple modules under example 4. And, therefore we in all in one, we attach each module along with which example it was traced from.

Reply:
Please create issue to update extractor to keep id same as in example if it was defined.
No duplicates of IDs.

Reply:
https://github.com/checkstyle/test-configs/issues/145